frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx

```diff
@@ -78,12 +78,19 @@ export const fuelCodeColDefs = (optionsData, errors, isCreate, canEdit) => [
     valueGetter: (params) => {
       if (params.data?.fuelCodePrefix?.prefix) {
         return params.data.fuelCodePrefix.prefix
-      } else if (params.data.prefixId) {
+      }else if (params.data.prefixId) {
         const selectedOption = optionsData?.fuelCodePrefixes?.find(
           (obj) => obj.fuelCodePrefixId === params.data.prefixId
         )
         return selectedOption.prefix
       }
+      const selectedOption = optionsData?.fuelCodePrefixes?.find(
+        (obj) => obj.prefix === params.data.prefix
+      )
+      if (params.data.prefix) {
```

Comment:
Looks like selectedOption could be undefined here due to the optional chaining. Should this be if (selectedOption) ?